lib/ansible/galaxy/data/default/collection/galaxy.yml.j2

```diff
@@ -0,0 +1,66 @@
+### REQUIRED
+
+# this can be a company/brand/organization or product namespace
+# under which all content lives
+namespace: {{ namespace }}
+
+
+# the designation of this specific collection
+name: {{ collection_name }}
+
+
+# semantic versioning compliant version designation
+version: 1.0.0
+
+# the filename for the readme file which can be either markdown (.md) or
+# reStructuredText (.rst)
+readme: README.md
+
+
+# a list of the collection's content authors
+# Ex: 'Full Name <email> (http://site) @nicks:irc/im/site#channel'
+authors:
+- {{ author }} <example@domain.com>
```

Comment:
We may want to use some filter to rip off starting quotes or urlencode them. otherwise, we're risking to generate invalid YAML, like:
```
- 'Broken Author <example@domain.com>
```

Reply:
The author is hardcoded to `your name` right now. If that changes we would then look at it but for now it's uneeded.